hack/make.sh

```diff
@@ -118,7 +118,8 @@ if [ "$AUTO_GOPATH" ]; then
 	if [ "$(go env GOOS)" = 'solaris' ]; then
 		# sys/unix is installed outside the standard library on solaris
 		# TODO need to allow for version change, need to get version from go
-		export GOPATH="${GOPATH}:/usr/lib/gocode/1.6.2"
+		export GO_VERSION=${GO_VERSION:-"1.6.3"}
```

Comment:
just to make it a bit more explicit (but wasn't sure if we could assume `GO_VERSION` was set already; it won't be if it's run outside a container

oh, ping @amitkris - FYI